index.html

```diff
@@ -239,13 +239,15 @@ <h1>Skills</h1>
                 </form>
             </section>
         </section>
+        <footer>
```

Comment:
- [ ] kindly use the correct different background image of the about me section for the mobile and desktop version.
your mobile
![Screenshot (645)_LI](https://user-images.githubusercontent.com/65298048/206551696-2c18f9ce-b53a-43c4-842a-4578c2c342e4.jpg)
figma mobile
![Screenshot (646)](https://user-images.githubusercontent.com/65298048/206551836-1a644733-f85b-4d12-ab56-ae2139c48c03.png)
your desktop
![Screenshot (647)](https://user-images.githubusercontent.com/65298048/206552221-1dde9f53-c56b-47a3-82d4-3323c0765847.png)
figma desktop
![Screenshot (648)](https://user-images.githubusercontent.com/65298048/206552365-20933f1d-c157-4fa7-be0a-784d68e8641a.png)